Source/Classes/_classes.json

```diff
@@ -2,62 +2,8 @@
    "outputFile": "vsteam.classes.ps1",
    "fileType": "classes",
    "files": [
-      "VSTeamVersions.ps1",
-      "VSTeamProjectCache.ps1",
-      "VSTeamProcessCache.ps1",
       "VSTeamDirectory.ps1",
```

Comment:
Just FYI - It is intentional to leave these two in, as they are base classes for the others and powershell won't work properly if they are "out of order".
The file list now gets filtered for unique values first.